src/PKSim.Core/Services/IndividualProteinTask.cs

```diff
@@ -125,41 +123,35 @@ protected void AddMucosaExpression(ISimulationSubject simulationSubject, string
       {
          foreach (var organ in simulationSubject.Organism.OrgansByName(CoreConstants.Organ.SmallIntestine, CoreConstants.Organ.LargeIntestine))
          {
-            var organMucosa = organ.Compartment(CoreConstants.Compartment.Mucosa);
-            foreach (var compartment in organMucosa.GetChildren<Compartment>().Where(c => c.Visible))
-            {
-               AddTissueParameters(compartment, moleculeName, CoreConstants.Groups.GI_MUCOSA);
-            }
+            var organMucosa = organ.Container(CoreConstants.Compartment.Mucosa);
+            organMucosa.GetChildren<Compartment>().Each(x => AddTissueParameters(x, moleculeName));
          }
       }
 
       protected void AddLumenExpression(ISimulationSubject simulationSubject, string moleculeName)
       {
          var lumen = simulationSubject.Organism.Organ(CoreConstants.Organ.Lumen);
-         foreach (var segment in lumen.Compartments.Where(c => c.Visible))
+         //Only visible compartments as we do not want to create RelExp for example in Feces
+         foreach (var segment in lumen.Compartments.Where(x => x.Visible))
```

Comment:
@Yuri05 Visible is required indeed. Otherwise compartmetn such as Feces are showing up